less/variables.less

```diff
@@ -145,7 +145,7 @@
 @brand-primary: #204f79;
 @brand-success: #596e31;
 @brand-info: #1d817a;
-@brand-warning: lighten(@id7-brand-orange, 10%);
```

Comment:
I'm wondering if this is the best way to effect this change. There's not much discernible difference between warning and danger now.